llmeval/launch_eval.py

```diff
@@ -24,6 +24,13 @@ def main():
         help="name of file containing models to evaluate (one per line).",
         required=False,
     )
+    parser.add_argument(
+        "--tasks_file",
+        type=str,
+        help="name of file containing tasks to evaluate (one per line).",
+        required=False,
+        default="tasks.txt",
```

Comment:
Would be nice to make it cwd independent:
```suggestion
        default=str(Path(__file__).parent / "tasks.txt"),
```
(assuming we import `from pathlib import Path` before)